src/content.rs

```diff
@@ -369,34 +456,44 @@ pub(crate) trait Container {
         )
     }
 
+    /// See `ContainedComments::add_block_comment`.
     fn add_block_comment(&mut self, comment: Comment) -> Result<(), Error> {
         self.contained_comments_mut().add_block_comment(comment)
     }
 
+    /// See `ContainedComments::has_pending_comments`.
     fn has_pending_comments(&self) -> bool {
         self.contained_comments().has_pending_comments()
     }
 
+    /// See `ContainedComments::take_pending_comments`.
     fn take_pending_comments(&mut self) -> Vec<Comment> {
         self.contained_comments_mut().take_pending_comments()
     }
 }
 
-pub(crate) struct Array {
-    /// The array items
+/// Represents a JSON5 array of items. During parsing, this object's state changes, as comments and
```

Comment:
In hindsight, I realize most of this comment is no longer relevant. (I think I wrote a lot of this before deciding to not make contained_comments public.)

In your next PR, can you also replace the struct Array comment here, and the struct Object comment with something like?

```rust
/// Represents a JSON5 array of items, plus any `trailing_comments` encountered after the last array item.
pub struct Array {

...
/// Represents a JSON5 object, containing property keys and values, plus any `trailing_comments` encountered after the last property.
pub struct Object {
```